tests/conftest.py

```
@@ -0,0 +1,50 @@
+    this fixture. This fixture is useful for testing the core client
+    or for integration tests that reload modules.
+    """
+    with patch("unraid_mcp.core.client.make_graphql_request", new_callable=AsyncMock) as mock:
+        yield mock
+
+
+def make_tool_fn(
+    module_path: str,
+    register_fn_name: str,
+    tool_name: str,
+) -> Any:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `make_tool_fn`

(ANN401)